include/zephyr/arch/arm/arch.h

```diff
@@ -99,13 +99,25 @@ enum k_fatal_error_reason_arch {
 	K_ERR_ARM_ALIGNMENT_FAULT,
 	K_ERR_ARM_BACKGROUND_FAULT,
 	K_ERR_ARM_PERMISSION_FAULT,
+	K_ERR_ARM_PERMISSION_FAULT_2nd_LEVEL,
```

Comment:
Maybe use caps for the whole define:
```suggestion
	K_ERR_ARM_PERMISSION_FAULT_2ND_LEVEL,
```
Ditto all others